config/locales/en.yml

```diff
@@ -1989,21 +1997,76 @@ en:
           lived_or_income_in_yonkers: Did you live or earn income in Yonkers at any time in 2023?
+          mfs_helper_description: Married filing separately is a tax filing status that is used by people who are legally married but choose to file separate returns.
+          mfs_helper_heading: What does “married filing separately” mean?
+          mfs_helper_no_if: 'Select no if:'
+          mfs_helper_no_mfj: You are married and filing jointly on both your federal and state returns.
+          mfs_helper_no_unmarried: You are not married.
+          mfs_helper_yes_federal_mfs: You filed “married filing separately” on your federal return.
+          mfs_helper_yes_if: 'Select yes if:'
+          mfs_helper_yes_state_mfs: You want to file your state return as “married filing separately”.
           subtitle: For the 2024 tax season, this tool will only be able to serve people with specific tax situations. Please answer the following questions to find out if you can use it.
           title: First, let's see if you can use this tool to file your taxes
       eligible:
+        az_unsupported:
+          other_credits_301_html: Nonrefundable credits listed on <a href="https://azdor.gov/sites/default/files/2023-03/FORMS_CREDIT_2022_301-SBI_f.pdf" target="_blank" rel="noopener nofollow">AZ Form 301</a>
+          other_credits_529_html: <a href="https://az529.gov/what-is-a-529-plan/az-tax-incentives/" target="_blank" rel="noopener nofollow">Arizona 529 plan contributions deduction</a>
+          other_credits_add_dependents: Adding dependents not claimed on the federal return
+          other_credits_change_filing_status: Change in filing status from federal to state return
+          other_credits_heading: Other credits and deductions not supported this year:"
+          other_credits_itemized_deductions: Itemized deductions
+          other_credits_qcos_html: <a href="https://azdor.gov/tax-credits/credits-contributions-qcos-and-qfcos" target="_blank" rel="noopener nofollow">Credit for Contributions to Qualifying Charitable Organizations</a>
+          other_credits_qfcos_html: <a href="https://azdor.gov/tax-credits/credits-contributions-qcos-and-qfcos" target="_blank" rel="noopener nofollow">Credit for Contributions to Qualifying Foster Care Charitable Organization</a>
+          property_tax_credit_age: Be 65 or older; or receive Supplemental Security Income
+          property_tax_credit_heading_html: '<a href="https://azdor.gov/individuals/income-tax-filing-assistance/tax-credits" target="_blank" rel="noopener nofollow">Property Tax Credit</a>. To qualify for this credit you must:
+
+            '
+          property_tax_credit_income: have total household income less than $3,751 if you live alone; or total household income less than $5,501 if you live with others; and
+          property_tax_credit_paid: have paid property taxes or rent on a home in Arizona during the year
```

Comment:
This seemed off to me and I looked in the story, these bullet points should actually be capitalized

```suggestion
          property_tax_credit_income: Have total household income less than $3,751 if you live alone; or total household income less than $5,501 if you live with others; and
          property_tax_credit_paid: Have paid property taxes or rent on a home in Arizona during the year
```

Reply:
hmm yeah...it's lower case in the content doc which is what i was going off of, thought that was supposed to be the source of truth for copy

Reply:
oh gosh, idk. here's the figma screenshot from the story. i guess don't change it for now and if someone really cares we'll fix it later.

![image](https://github.com/codeforamerica/vita-min/assets/122047394/1a7c3741-570f-417a-9f42-d5f51f97105d)